README.md

```diff
@@ -6,6 +6,45 @@ SDL bindings for Bare.
 npm i bare-sdl
 ```
 
+## Documentation
+
+### Window
+
+The Window API provides functionality to create SDL windows.
+
+```javascript
+const window = new SDLWindow(title, width, height, [flags])
```

Comment:
```suggestion
const window = new SDLWindow(title, width, height[, flags])
```

Reply:
Oh yeah, I still miss this syntax thanks :)